gateway/1.8/modules/ROOT/pages/_partials/task-tls-config.adoc

```diff
@@ -14,6 +14,11 @@ Flex Gateway supports the following:
 
 //end::intro2[]
 
+//tag::defaultOutboundTLS[]
+By default, Flex Gateway supports outbound TLS to communicate with upstream services that require a secure communication channel. For default outbound communication, Flex Gateway uses only TLS 1.2. To find the default TLS 1.2 ciphers, see <<supported-ciphers>>. To find the ciphers, see <<supported-ciphers>>. Applying a TLS context to an upstream overrides the default TLS context for that upstream. Because the default ciphers might change for later Flex Gateway versions, you can apply a TLS context to an upstream to ensure it remains the same.
```

Comment:
```suggestion
By default, Flex Gateway supports outbound TLS to communicate with upstream services that require a secure communication channel. For default outbound communication, Flex Gateway uses only TLS 1.2. To find the default TLS 1.2 ciphers, see <<supported-ciphers>>. To find the ciphers, see <<supported-ciphers>>. Applying a TLS context to an upstream overrides the default TLS context for that upstream. Because the default ciphers may change for later Flex Gateway versions, you can apply a TLS context to an upstream to ensure it remains the same.
```